liquidctl/cli.py

```diff
@@ -28,13 +28,17 @@
   --time-off <value>          Time to wait with the LED turned off (seconds)
   --alert-threshold <number>  Threshold temperature for a visual alert (°C)
   --alert-color <color>       Color used by the visual high temperature alert
+  --direction <string>        If the pattern should move forward or backwords. [default: forward]
+  --start-led <number>        The first led to start the effect at (0 offset)
+  --num-leds <number>         The number of LED's the effect shoud apply to
 
 Other device options:
   --single-12v-ocp            Enable single rail +12V OCP
   --pump-mode <mode>          Set the pump mode (certain Corsair coolers)
   --legacy-690lc              Use Asetek 690LC in legacy mode (old Krakens)
   --non-volatile              Store on non-volatile controller memory
   --unsafe <features>         Comma-separated bleeding-edge features to enable
+  --temp-sensor <number>      The tempature sensor number for the Commander Pro 
```

Comment:
- typo in description: s/tempature/temperature/
- avoid abbreviation: `--temperature-sensor`